File: C/pointers/b_array/p6.c
#include<stdio.h>
int main(){
	int a[]={100,200,300};
	int b[]={400,200,300};
	int c[]={900,200,300};


	int* (*arr)[5]={a,b,c};




	for(int i=0;i<3;i++){
		printf("%d\n",***(arr+i));

	}





}

File: java/enum/p6.java
enum Level {
  LOW(-1),
  MEDIUM(0),
  HIGH(1),
  VERYHIgh(2);

  private int value;

  Level(int value){
      this.value = value;
  }

  // public int getValue(){
  //     return this.value;
  // }

  public static Level valueOf(){
    return this.value;
  
  }


}


class Test{
	public static void main(String args[]){

    Level arr[] = Level.values();


    for(Level m : arr){
      System.out.println(Level.valueOf(m));

	}
    }
}
